src/System.Private.CoreLib/shared/System/Globalization/CultureData.cs

```diff
@@ -2328,17 +2330,17 @@ internal void GetNFIValues(NumberFormatInfo nfi)
         {
             if (GlobalizationMode.Invariant || IsInvariantCulture)
             {
-                nfi._positiveSign = _sPositiveSign;
-                nfi._negativeSign = _sNegativeSign;
+                nfi._positiveSign = _sPositiveSign!;
+                nfi._negativeSign = _sNegativeSign!;
 
-                nfi._numberGroupSeparator = _sThousandSeparator;
-                nfi._numberDecimalSeparator = _sDecimalSeparator;
+                nfi._numberGroupSeparator = _sThousandSeparator!;
+                nfi._numberDecimalSeparator = _sDecimalSeparator!;
```

Comment:
I see that right after this we check for some null values in some of this fields, should we make them nullable instead o using ! ?

Reply:
actually they are not nullable, I didn't annotate that file yet but I'm trying to do that without doing refactoring (CultureData seem a bit special when it comes to structuring the code so likely will have to do it)

Reply:
actually that was CalendarData which was not marked as nullable - will check again why I `!`ed them

Reply:
they cannot be null but I'd have to refactor this code to do it in a way which doesn't require banging everything

Reply:
Then leave the bang. The questions that I have is, then should we, be checking for null after this assignments? That is what confused me, we bang them, but we check for null right after.

Reply:
yes we do, because they can be null while in transient state but for consumer of this class they are never null (that's why I was talking about refactor)